05LessonFive/Checkpoint1/index.html

```diff
@@ -1,10 +1,33 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title>Landscape Design</title>
-  <link rel="stylesheet" href="css/style.css">
+    <title>Landscape Design</title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
+        <table>
+            <tr>
+              <div id="left">
+                <td>
+                    <img src="img/berries.jpg" />
+                </td>
+    </div>
+    <td>
+        <div id="right">
+            <h2>Berries</h2>
+            </p>Etiam pharetra, erat sed fermentum feugiat, velit mauris egesta quam, ut aliquam massa nisl neque. Suspendisse in orci enim</p>
+            <h3><a href="#">Read More</a></h3>
+        </div>
+    </td>
+
+
+    </tr>
+    </table>
+    </section>
+
 
 </body>
+
 </html>
```

Comment:
Let's look at this during lab, it doesn't look like the code you have here is what is being rendered on the page.